src/integration/common/a11y.js

```diff
@@ -1,10 +1,9 @@
-export default ({ skipToContentText, headlineText }) => {
-  [amp, canonical].forEach((page) => {
+export default ({ skipToContentText }) => {
+  [amp, canonical].forEach(page => {
     describe(`And using ${page.platform}`, () => {
       it('I can see a skip to content link that links to the main content of the page', () => {
         const skipToContentEl = page.getByText(skipToContentText);
-        const mainContentEl = page.getByText(headlineText);
-
+        const mainContentEl = page.document.querySelector('h1');
```

Comment:
Good switch.
we might need to refactor https://github.com/bbc/simorgh/blob/2784bbaf284372b0862b9057cb335c0da16ff6e4/src/integration/pages/articlePage/pidgin.test.js#L102-L107 and https://github.com/bbc/simorgh/blob/2784bbaf284372b0862b9057cb335c0da16ff6e4/src/integration/pages/liveRadioPage/amharic.test.js#L106-L111 and all other tests under `articles` and `liveRadioPage` directories since they're passing in a `headlineText`

Reply:
Done.